assets/i18n/tr.json

```diff
@@ -0,0 +1,299 @@
+  "I18N_SIGNUP_UPDATE_WARNING": "Lütfen Kullanım Koşullarımızı son zamanlarda güncellediğimizi unutmayın.",
+  "I18N_SIGNUP_USERNAME": "Kullanıcı Adı",
+  "I18N_SIGNUP_USERNAME_EXPLANATION": "Kullanıcı adın katkılarının yanında gösterilecektir.",
+  "I18N_SIGNUP_WAIVER_OBJECTIVE": "Atıfta bulunma şartından feragat etmek (BY) şu anlama gelir, eğer biri çalışmanızı tekrar kullanırsa, kullananların yazara atıfta bulunmasına gerek yoktur. Bununla birlikte, konu anlatımlarına yaptığınız bireysel katkılar sitede konu anlatımları sistem değişiklik kaydında gözükecektir ve konu anlatımlarını tekrar kullananlar bu sayfaya link eklemeye teşvik edilecektir (fakat gerekli değil).",
+  "I18N_SIGNUP_WHY_LICENSE": "Neden CC-BY-SA?",
+  "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION": "Oppia dersleri, konu anlatımları, statik video ve metinlerden daha kapsamlı öğrenme deneyimler, sağlayarak, kullanıcıların yaparak öğrenmesine yardımcı olur.",
+  "I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION": "Oppia ücretsiz, tamamen etkileşimli konu anlatımları içeren, açık kaynak kodlu bir öğrenme platformudur.  Maalesef, Oppia sağlıklı bir şekilde çalışması için tarayıcınızdaki JavaScript eklentisinin aktive edilmesini gerektirmektedir ve tarayıcınızda JavaScript kapalı durumda.  Eğer JavaScript'i çalıştırmak istiyorsanız, <a href=\"<[hrefUrl]>\">buraya tıklayın.</a>",
+  "I18N_SPLASH_JAVASCRIPT_ERROR_THANKS": "Teşekkürler.",
+  "I18N_SPLASH_JAVASCRIPT_ERROR_TITLE": "Tarayıcınızda JavaScript olmasına ihtiyacımız var.",
+  "I18N_SPLASH_PAGE_TITLE": "Oppia: Öğret, Öğren, Keşfet",
+  "I18N_SPLASH_SECOND_EXPLORATION_DESCRIPTION": "Konu anlatımlarını oluşturması basit. Konu anlatımları bireysel öğrenci dönütleri ve dünya genelindeki öğrenenlerin tecrübeleri temelinde kolaylıkla uyarlanabilir.",
+  "I18N_SPLASH_SITE_FEEDBACK": "Site Dönütü",
+  "I18N_SPLASH_SUBTITLE": "Oppia, eğiten ve güdüleyen etkileşimli dersler oluşturmayı kolay hale getirir.",
+  "I18N_SPLASH_THIRD_EXPLORATION_DESCRIPTION": " Oppia, sadece hayal gücünle sınırlandırılmış olan, geniş bir konu aralığında konu anlatımları oluşturmana ve paylaşmana izin verir.",
+  "I18N_SPLASH_TITLE": "Kitapların dışında düşün.",
+  "I18N_TOPNAV_ABOUT": "Hakkında",
+  "I18N_TOPNAV_ABOUT_OPPIA": "Hakkımızda - Oppia",
+  "I18N_TOPNAV_ADMIN_PAGE": "Yönetici Sayfası",
+  "I18N_TOPNAV_BLOG": "Blog",
+  "I18N_TOPNAV_CONTACT_US": "Bize Ulaşın",
+  "I18N_TOPNAV_DASHBOARD": "Oluşturucu Vitrini",
+  "I18N_TOPNAV_DONATE": "Bağış Yap",
+  "I18N_TOPNAV_FORUM": "Forum",
+  "I18N_TOPNAV_GET_STARTED": "Başla",
+  "I18N_TOPNAV_LIBRARY": "Kütüphane",
+  "I18N_TOPNAV_LOGOUT": "Çıkış",
+  "I18N_TOPNAV_MODERATOR_PAGE": "Moderatör Sayfası",
+  "I18N_TOPNAV_NOTIFICATIONS": "Bildirimler",
+  "I18N_TOPNAV_PREFERENCES": "Tercihler",
+  "I18N_TOPNAV_SIGN_IN": "Giriş Yap",
+  "I18N_TOPNAV_TEACH_WITH_OPPIA": "Oppia ile Öğret"
+}
```

Comment:
Er, nope. I meant after the very last line, so that the red mark at the end doesn't show when you view the list of files changed on github.

Everything else looks good, thanks!